### PR TITLE
Add tex templates dropdown

### PR DESCRIPTION
**Pull request by universvm**

### CODE REVIEW

**Comment by universvm** [comment]
@mokeyish what do you think?

**Comment by mokeyish** [comment]
No, you can't just copy your code to override this branch.

File: src/ui/SettingTab.tsx [hunk 1]
```diff
@@ -17,39 +17,18 @@ import type { Lang } from '../lang';
...
   const { plugin, lang } = props;
   const [settings, setSettings0] = createStore(plugin.settings);
   const [pandocVersion, setPandocVersion] = createSignal<string>();
-  const envVars = createMemo(() => Object.entries(getPlatformValue(settings.env) ?? {}).map(([n, v]) => `${n}="${v}"`).join('\n'));
   const setSettings: typeof setSettings0 = (...args: unknown[]) => {
     (setSettings0 as ((...args: unknown[]) => void))(...args);
     plugin.saveSettings();
   };
-  const setEnvVars = (envItems: string) => {
-    try {
```
[the remark this answers is not in the document]
Comment (universvm) [reply src/ui/SettingTab.tsx:34]: I don't understand how these changes keep coming up sorry.

**Comment by universvm** [comment]
@mokeyish 

I pulled from main and applied the changes. Could it be that this is why you say your code was deleted?

**Comment by universvm** [comment]
> No, you can't just copy your code to override this branch.

I don't understand what I did wrong... Can you help out?

**Comment by universvm** [comment]
How can these changes be deleted if I merged dev into my branch?

**Comment by mokeyish** [comment]
> @mokeyish
> 
> I pulled from main and applied the changes. Could it be that this is why you say your code was deleted?


You can see the tab `File changes` above. The red means code removed, the green means code added.


In theory, you only need to change src/export_templates.ts for ts/tsx files.

**Comment by universvm** [comment]
Ok, please can you wait before you do any more changes? I have not deleted anything...

File: src/ui/SettingTab.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mokeyish) [reply src/ui/SettingTab.tsx:34]: I can help you resolve the conflicts.

It seems that you are not very familiar with git and can't resolve conflicts well.  

I'm worried, I resolved conflicts for you, and you overwrote it again.

**Comment by mokeyish** [comment]
I've help you resolve the conflicts in the branch https://github.com/mokeyish/obsidian-enhancing-export/tree/templates_dropdown

**Comment by universvm** [comment]
I am not sure what happened. I have now fixed used a different branch.

**Pull request closed without merging**